tests/src/test/v1.2/advanced/fdc3.open.ts

```diff
@@ -317,17 +320,18 @@ const waitForContext = (
               );
               if (executionListener) executionListener.unsubscribe();
               resolve(context);
+              return;
```

Comment:
is this needed?

Reply:
`return` prevents the execution of any other code within that function. It's generally good practice to return after resolving/rejecting promises to avoid unwanted code execution

Reply:
Except it doesn't if that execution occurs inside other async functions, promise constructors etc. and it can give rise to a false sense of security when reviewing code.

Many consider it a waste of space ;-)

I don't mind this one as it's embedded in a set of if/else clauses and can help with readability (as long as you not lulled into ignoring nesting inside promise constructors and other async function calls) - although it is in fact changing nothing at all. Other developers argue that you *should* be interpreting those clauses anyway and not skipping over because you saw a return statement. 

However, tacking them on at the end of functions (where there is no further code afterwards) is generally considered poor practice.